src/routes/(console)/organization-[organization]/usage/[[invoice]]/+page.svelte

```diff
@@ -119,36 +119,41 @@
                 <ProgressBarBig
                     currentUnit={currentHumanized.unit}
                     currentValue={currentHumanized.value}
-                    maxUnit="GB"
-                    maxValue={max.toString()}
+                    maxUnit=""
```

Comment:
feel free to make it an optional param 👍🏻

Reply:
Adjusted